Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandSetLabel.java

```diff
@@ -69,11 +74,10 @@ public void run(Map<String, String> parameters) throws UserErrorMessage
 		
 		// Assign new label.
 		try {
-			String label = parameters.get("label");
 			if (label==null || label.trim().equals(""))
 				item.setProperty("label", null);
 			else
-				item.setProperty("label", label.trim());
+				item.setProperty("label", label.replace("'", "`").trim());
```

Comment:
I often add these sorts of replacements without really knowing if it's an issue or not. Just fyi. This might be justified, just saying.